lib/ocpp/common/database/database_connection.cpp

```diff
@@ -28,8 +29,11 @@ class DatabaseTransaction : public DatabaseTransactionInterface {
     }
 
     void commit() override {
-        this->database.execute_statement("COMMIT TRANSACTION");
+        const auto retval = this->database.execute_statement("COMMIT TRANSACTION");
```

Comment:
I am wondering if `execute_statement` should not throw if it fails as well. Maybe that is for another PR then.

Reply:
I was thinking the same but I left it for now since its not part of any public api. No strong opinion here...